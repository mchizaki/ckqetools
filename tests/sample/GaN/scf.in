&control
  calculation   = 'scf'
  restart_mode  = 'from_scratch'
  prefix        = 'GaN'
  outdir        = 'out_GaN'
/

&system
  ibrav   = 4
  a       = 3.18893040
  c       = 5.19235725
  nat     = 4
  ntyp    = 2
  ecutwfc = 100
  ecutrho = 400
/

&electrons
/

k_points {automatic}
4 4 4 1 1 1

ATOMIC_SPECIES
  Ga   69.72300  Ga.pbe-dn-kjpaw_psl.1.0.0.UPF
   N   14.00650  N.pbe-n-kjpaw_psl.1.0.0.UPF

ATOMIC_POSITIONS {crystal}
Ga   0.666666666666667   0.333333333333333   0.499083710000000
Ga   0.333333333333333   0.666666666666667   0.999083710000000
 N   0.666666666666667   0.333333333333333   0.875916290000000
 N   0.333333333333333   0.666666666666667   0.375916290000000

