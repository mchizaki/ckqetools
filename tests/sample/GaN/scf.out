
     Program PWSCF v.7.2 starts on 10Jul2024 at 11:55:22 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on      96 processor cores
     Number of MPI processes:                96
     Threads/MPI process:                     1

     MPI processes distributed on     1 nodes
     89123 MiB available memory on the printing compute node when the environment starts
 
     Waiting for input...
     Reading input from standard input

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  4
     file Ga.pbe-dn-kjpaw_psl.1.0.0.UPF: wavefunction(s)  4S renormalized
     file N.pbe-n-kjpaw_psl.1.0.0.UPF: wavefunction(s)  2S renormalized
 
     K-points division:     npool     =       3
     R & G space division:  proc/nbgrp/npool/nimage =      32
     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min          31      31      9                 1305     1305     205
     Max          32      32     10                 1312     1312     210
     Sum        1003    1003    295                41829    41829    6627
 
     Using Slab Decomposition
 


     bravais-lattice index     =            4
     lattice parameter (alat)  =       6.0262  a.u.
     unit-cell volume          =     308.5900 (a.u.)^3
     number of atoms/cell      =            4
     number of atomic types    =            2
     number of electrons       =        36.00
     number of Kohn-Sham states=           18
     kinetic-energy cutoff     =     100.0000  Ry
     charge density cutoff     =     400.0000  Ry
     scf convergence threshold =      1.0E-06
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation= SLA  PW   PBX  PBC
                           (   1   4   3   4   0   0   0)

     celldm(1)=   6.026205  celldm(2)=   0.000000  celldm(3)=   1.628244
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (  -0.500000   0.866025   0.000000 )  
               a(3) = (   0.000000   0.000000   1.628244 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.577350  0.000000 )  
               b(2) = (  0.000000  1.154701  0.000000 )  
               b(3) = (  0.000000  0.000000  0.614159 )  


     PseudoPot. # 1 for Ga read from file:
     /home/ee5/local/espresso/pseudo/Ga.pbe-dn-kjpaw_psl.1.0.0.UPF
     MD5 check sum: 9ec968fb6aa164ad922e5aaa76dd79f0
     Pseudo is Projector augmented-wave + core cor, Zval = 13.0
     Generated using &quot;atomic&quot; code by A. Dal Corso  v.6.3
     Shape of augmentation charge: PSQ
     Using radial grid of 1205 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2
     Q(r) pseudized with 0 coefficients 


     PseudoPot. # 2 for N  read from file:
     /home/ee5/local/espresso/pseudo/N.pbe-n-kjpaw_psl.1.0.0.UPF
     MD5 check sum: c8d345807381b1d4df424700dd57e850
     Pseudo is Projector augmented-wave + core cor, Zval =  5.0
     Generated using &quot;atomic&quot; code by A. Dal Corso  v.6.3
     Shape of augmentation charge: PSQ
     Using radial grid of 1085 points,  4 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
     Q(r) pseudized with 0 coefficients 


     atomic species   valence    mass     pseudopotential
        Ga            13.00    69.72300     Ga( 1.00)
        N              5.00    14.00650     N ( 1.00)

     12 Sym. Ops. (no inversion) found ( 6 have fractional translation)



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Ga  tau(   1) = (   0.5000000   0.2886751   0.8126301  )
         2           Ga  tau(   2) = (  -0.0000000   0.5773503   1.6267522  )
         3           N   tau(   3) = (   0.5000000   0.2886751   1.4262056  )
         4           N   tau(   4) = (  -0.0000000   0.5773503   0.6120835  )

     number of k points=    12
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.1250000   0.2165064   0.0767698), wk =   0.1250000
        k(    2) = (   0.1250000   0.2165064   0.2303094), wk =   0.1250000
        k(    3) = (   0.1250000   0.5051815   0.0767698), wk =   0.2500000
        k(    4) = (   0.1250000   0.5051815   0.2303094), wk =   0.2500000
        k(    5) = (   0.1250000  -0.3608439   0.0767698), wk =   0.2500000
        k(    6) = (   0.1250000  -0.3608439   0.2303094), wk =   0.2500000
        k(    7) = (   0.1250000  -0.0721688   0.0767698), wk =   0.1250000
        k(    8) = (   0.1250000  -0.0721688   0.2303094), wk =   0.1250000
        k(    9) = (   0.3750000   0.6495191   0.0767698), wk =   0.1250000
        k(   10) = (   0.3750000   0.6495191   0.2303094), wk =   0.1250000
        k(   11) = (   0.3750000  -0.2165064   0.0767698), wk =   0.1250000
        k(   12) = (   0.3750000  -0.2165064   0.2303094), wk =   0.1250000

     Dense  grid:    41829 G-vectors     FFT dimensions: (  40,  40,  64)

     Estimated max dynamical RAM per process >       5.63 MB

     Estimated total dynamical RAM >     520.37 MB

     Check: negative core charge=   -0.000001

     Initial potential from superposition of free atoms

     starting charge      35.9941, renormalised to      36.0000
     Starting wfcs are   26 randomized atomic wfcs
     Checking if some PAW data can be deallocated... 

     total cpu time spent up to now is        0.9 secs

     Self-consistent Calculation

     iteration #  1     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  1.3

     total cpu time spent up to now is        1.2 secs

     total energy              =    -611.64770774 Ry
     estimated scf accuracy    <       1.13002059 Ry

     iteration #  2     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.14E-03,  avg # of iterations =  5.8

     total cpu time spent up to now is        1.6 secs

     total energy              =    -612.12423349 Ry
     estimated scf accuracy    <       1.30509890 Ry

     iteration #  3     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.14E-03,  avg # of iterations =  2.8

     total cpu time spent up to now is        1.9 secs

     total energy              =    -612.36728413 Ry
     estimated scf accuracy    <       0.02866162 Ry

     iteration #  4     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  7.96E-05,  avg # of iterations =  8.0

     total cpu time spent up to now is        2.4 secs

     total energy              =    -612.44791317 Ry
     estimated scf accuracy    <       0.01627533 Ry

     iteration #  5     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.52E-05,  avg # of iterations =  6.0

     total cpu time spent up to now is        2.9 secs

     total energy              =    -612.41061460 Ry
     estimated scf accuracy    <       0.34770064 Ry

     iteration #  6     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.52E-05,  avg # of iterations =  5.0

     total cpu time spent up to now is        3.3 secs

     total energy              =    -612.42445782 Ry
     estimated scf accuracy    <       0.04222698 Ry

     iteration #  7     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.52E-05,  avg # of iterations =  2.3

     total cpu time spent up to now is        3.6 secs

     total energy              =    -612.43017407 Ry
     estimated scf accuracy    <       0.00135951 Ry

     iteration #  8     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.78E-06,  avg # of iterations =  7.7

     total cpu time spent up to now is        4.1 secs

     total energy              =    -612.43165310 Ry
     estimated scf accuracy    <       0.00131383 Ry

     iteration #  9     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.65E-06,  avg # of iterations =  1.2

     total cpu time spent up to now is        4.4 secs

     total energy              =    -612.43162749 Ry
     estimated scf accuracy    <       0.00146061 Ry

     iteration # 10     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.65E-06,  avg # of iterations =  1.0

     total cpu time spent up to now is        4.9 secs

     total energy              =    -612.43165257 Ry
     estimated scf accuracy    <       0.00054756 Ry

     iteration # 11     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.52E-06,  avg # of iterations =  1.0

     total cpu time spent up to now is        5.2 secs

     total energy              =    -612.43156747 Ry
     estimated scf accuracy    <       0.00073029 Ry

     iteration # 12     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.52E-06,  avg # of iterations =  2.0

     total cpu time spent up to now is        5.5 secs

     total energy              =    -612.43162435 Ry
     estimated scf accuracy    <       0.00003127 Ry

     iteration # 13     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  8.68E-08,  avg # of iterations =  9.9

     total cpu time spent up to now is        5.9 secs

     total energy              =    -612.43168051 Ry
     estimated scf accuracy    <       0.00003888 Ry

     iteration # 14     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  8.68E-08,  avg # of iterations =  1.1

     total cpu time spent up to now is        6.3 secs

     total energy              =    -612.43167702 Ry
     estimated scf accuracy    <       0.00003966 Ry

     iteration # 15     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  8.68E-08,  avg # of iterations =  1.0

     total cpu time spent up to now is        6.5 secs

     total energy              =    -612.43167250 Ry
     estimated scf accuracy    <       0.00002718 Ry

     iteration # 16     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  7.55E-08,  avg # of iterations =  1.6

     total cpu time spent up to now is        6.8 secs

     total energy              =    -612.43168039 Ry
     estimated scf accuracy    <       0.00001101 Ry

     iteration # 17     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.06E-08,  avg # of iterations =  6.6

     total cpu time spent up to now is        7.2 secs

     total energy              =    -612.43168704 Ry
     estimated scf accuracy    <       0.00007025 Ry

     iteration # 18     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.06E-08,  avg # of iterations =  1.1

     total cpu time spent up to now is        7.5 secs

     total energy              =    -612.43168815 Ry
     estimated scf accuracy    <       0.00006359 Ry

     iteration # 19     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.06E-08,  avg # of iterations =  1.1

     total cpu time spent up to now is        7.8 secs

     total energy              =    -612.43167692 Ry
     estimated scf accuracy    <       0.00006809 Ry

     iteration # 20     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.06E-08,  avg # of iterations =  2.0

     total cpu time spent up to now is        8.1 secs

     total energy              =    -612.43166977 Ry
     estimated scf accuracy    <       0.00003503 Ry

     iteration # 21     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.06E-08,  avg # of iterations =  3.4

     total cpu time spent up to now is        8.4 secs

     total energy              =    -612.43166838 Ry
     estimated scf accuracy    <       0.00000931 Ry

     iteration # 22     ecut=   100.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.59E-08,  avg # of iterations =  4.9

     total cpu time spent up to now is        8.7 secs

     End of self-consistent calculation

          k = 0.1250 0.2165 0.0768 (  5227 PWs)   bands (ev):

    -5.8443  -5.2931  -3.7159  -3.7070  -3.6503  -3.6398  -3.5554  -3.4985
    -3.4983  -3.4621  -3.3911  -2.0069   3.5008   6.5339   6.9062   7.9204
     9.0601   9.1484

          k = 0.1250 0.2165 0.2303 (  5216 PWs)   bands (ev):

    -5.6155  -5.3833  -3.7030  -3.6857  -3.6722  -3.6562  -3.5571  -3.5180
    -3.4560  -3.4340  -3.2584  -2.7235   4.7543   6.0847   7.2481   8.0416
     8.5083   8.6297

          k = 0.1250 0.5052 0.0768 (  5209 PWs)   bands (ev):

    -5.3632  -5.2128  -3.7115  -3.6680  -3.5753  -3.5482  -3.5188  -3.4808
    -3.4306  -3.4032  -2.3673  -1.6233   3.4044   4.4425   5.7037   6.4793
     7.7898   8.1058

          k = 0.1250 0.5052 0.2303 (  5208 PWs)   bands (ev):

    -5.2820  -5.2200  -3.6962  -3.6598  -3.6347  -3.5606  -3.5085  -3.4714
    -3.4255  -3.4173  -2.1974  -1.8868   3.2380   3.6933   6.5566   7.3429
     7.4969   8.2983

          k = 0.1250-0.3608 0.0768 (  5215 PWs)   bands (ev):

    -5.5381  -5.2356  -3.7008  -3.6795  -3.6052  -3.5721  -3.5334  -3.5031
    -3.4546  -3.4151  -3.0591  -1.6982   3.8373   5.2366   5.7531   6.9539
     8.4424   8.6087

          k = 0.1250-0.3608 0.2303 (  5203 PWs)   bands (ev):

    -5.3977  -5.2730  -3.6868  -3.6621  -3.6406  -3.6005  -3.5328  -3.5160
    -3.4367  -3.4261  -2.8114  -2.2762   4.1164   4.7948   6.8512   7.3063
     7.8724   8.4125

          k = 0.1250-0.0722 0.0768 (  5231 PWs)   bands (ev):

    -6.0856  -5.3507  -3.7485  -3.7317  -3.6986  -3.6753  -3.6677  -3.5080
    -3.4996  -3.4340  -3.4059  -2.2132   3.1453   7.6557   8.3076   8.3968
     9.3953   9.5051

          k = 0.1250-0.0722 0.2303 (  5226 PWs)   bands (ev):

    -5.8011  -5.4863  -3.7299  -3.7190  -3.7050  -3.6923  -3.5881  -3.4758
    -3.4670  -3.4284  -3.4160  -2.9682   4.7829   6.7868   8.1020   8.7038
     8.7839   9.1618

          k = 0.3750 0.6495 0.0768 (  5197 PWs)   bands (ev):

    -5.2802  -5.2223  -3.7398  -3.6060  -3.5601  -3.5308  -3.5204  -3.4861
    -3.4591  -3.4303  -1.9287  -1.6334   3.7853   4.3474   4.9196   6.5896
     7.0553   7.3669

          k = 0.3750 0.6495 0.2303 (  5202 PWs)   bands (ev):

    -5.2303  -5.2065  -3.6712  -3.6397  -3.6289  -3.5682  -3.4939  -3.4797
    -3.4512  -3.4474  -1.8448  -1.7186   3.4506   3.6694   5.3822   6.2689
     7.8089   8.1805

          k = 0.3750-0.2165 0.0768 (  5211 PWs)   bands (ev):

    -5.4644  -5.2233  -3.7052  -3.6750  -3.5996  -3.5534  -3.5286  -3.4886
    -3.4408  -3.3902  -2.8310  -1.6577   3.5443   4.8051   5.5075   7.2239
     8.2381   8.7673

          k = 0.3750-0.2165 0.2303 (  5197 PWs)   bands (ev):

    -5.3489  -5.2496  -3.7112  -3.6690  -3.6296  -3.5674  -3.5196  -3.4772
    -3.4311  -3.4112  -2.5994  -2.1264   3.5527   4.1569   6.8992   7.6323
     8.2223   8.2656

     highest occupied level (ev):     9.5051

!    total energy              =    -612.43167175 Ry
     total all-electron energy =     -7995.358590 Ry
     estimated scf accuracy    <       0.00000034 Ry

     The total energy is the sum of the following terms:
     one-electron contribution =    -181.05226996 Ry
     hartree contribution      =     130.20203765 Ry
     xc contribution           =     -88.48090214 Ry
     ewald contribution        =    -233.17481799 Ry
     one-center paw contrib.   =    -239.92571930 Ry

     convergence has been achieved in  22 iterations

     Writing all to output data dir out_GaN/GaN.save/
 
     init_run     :      0.37s CPU      0.51s WALL (       1 calls)
     electrons    :      6.40s CPU      7.84s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.02s CPU      0.04s WALL (       1 calls)
     potinit      :      0.18s CPU      0.24s WALL (       1 calls)
     hinit0       :      0.12s CPU      0.15s WALL (       1 calls)

     Called by electrons:
     c_bands      :      2.25s CPU      2.83s WALL (      22 calls)
     sum_band     :      0.59s CPU      0.72s WALL (      22 calls)
     v_of_rho     :      0.18s CPU      0.25s WALL (      23 calls)
     newd         :      0.21s CPU      0.26s WALL (      23 calls)
     PAW_pot      :      3.16s CPU      3.76s WALL (      23 calls)
     mix_rho      :      0.07s CPU      0.08s WALL (      22 calls)

     Called by c_bands:
     init_us_2    :      0.03s CPU      0.03s WALL (     180 calls)
     init_us_2:cp :      0.03s CPU      0.03s WALL (     180 calls)
     cegterg      :      1.90s CPU      2.38s WALL (      88 calls)

     Called by *egterg:
     cdiaghg      :      0.23s CPU      0.24s WALL (     392 calls)
     h_psi        :      1.42s CPU      1.83s WALL (     396 calls)
     s_psi        :      0.02s CPU      0.02s WALL (     396 calls)
     g_psi        :      0.00s CPU      0.00s WALL (     304 calls)

     Called by h_psi:
     h_psi:calbec :      0.09s CPU      0.12s WALL (     396 calls)
     vloc_psi     :      1.31s CPU      1.68s WALL (     396 calls)
     add_vuspsi   :      0.02s CPU      0.02s WALL (     396 calls)

     General routines
     calbec       :      0.11s CPU      0.15s WALL (     484 calls)
     fft          :      0.14s CPU      0.19s WALL (     253 calls)
     ffts         :      0.03s CPU      0.04s WALL (      22 calls)
     fftw         :      1.44s CPU      1.84s WALL (   11544 calls)
 
     Parallel routines
 
     PWSCF        :      7.22s CPU      8.90s WALL

 
   This run was terminated on:  11:55:31  10Jul2024            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
